etna/metrics/metrics.py

```diff
@@ -68,6 +69,35 @@ def greater_is_better(self) -> bool:
         return False
 
 
+class RMSE(Metric):
+    """Root mean squared error metric with multi-segment computation support.
+
+    .. math::
+        RMSE(y\_true, y\_pred) = \\sqrt\\frac{\\sum_{i=0}^{n-1}{(y\_true_i - y\_pred_i)^2}}{n}
+
+    Notes
+    -----
+    You can read more about logic of multi-segment metrics in Metric docs.
+    """
+
+    def __init__(self, mode: str = MetricAggregationMode.per_segment, **kwargs):
+        """Init metric.
+
+        Parameters
+        ----------
+        mode: 'macro' or 'per-segment'
+            metrics aggregation mode
+        kwargs:
+            metric's computation arguments
+        """
+        super().__init__(mode=mode, metric_fn=rmse, **kwargs)
```

Comment:
In fact it is not necessary to create the separate functional metric for this case. Try to use partial module to reuse the `mse` metric.
1. `partial` usage example [link](https://www.geeksforgeeks.org/partial-functions-python/)
2. See the `mse` flag `squared` in the sklearn docs [link](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html)(we use their implementation)

Add the definition of `rmse` to the `__init__` file [link](https://github.com/tinkoff-ai/etna/blob/master/etna/metrics/__init__.py)

Reply:
Thank you for the feedback, I thought about using partial just for `RMSE` class, but I find it more consistent to define `rmse` in `functional_metrics.py`, which under hood uses `mse` with `squared=False` from sklearn. 
In case functional metric is not needed, I can just use partial. So which option do you think is best?

Reply:
May you use partial, the less code the better)))

Reply:
Ok, done, I had to put it to `functional_metrics.py` to prevent circular import.